.github/workflows/test.yml

```diff
@@ -0,0 +1,29 @@
+name: Continuous Integration
+
+on: [pull_request]
+
+jobs:
+  build:
+    name: ci
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        scala: [2.12.12, 2.11.12]
+    container:
+      image: ucbbar/chisel3-tools
+      options: --user github --entrypoint /bin/bash
+    env:
+      CONTAINER_HOME: /home/github
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Scala
+        uses: olafurpg/setup-scala@v10
```

Comment:
it might make sense to specify the JDK version to make it clear what the supported version is

Reply:
Sure, I was leaving it as the default which is Java8 but it's not a bad idea to be specific.

We probably ought to do a matrix of Java versions, maybe Java 8, 11, and 15. Before doing that though, I would want to have a way to use treadle for testing (instead of Verilator) to reduce CI load (running Verilator on just 1 out of the matrix)